python/gtfsToTimetableApi/GtfsTimeTable.py

```diff
@@ -20,6 +20,7 @@
     717: "FÖRBESTÄLLNINGSTRAFIK",
```

Comment:
This row can be removed since it is no longer part of the GTFS specification, and has been replaced by route type 1501